### PR TITLE
Add LoRA parameter to test_lm_eval_correctness

### PR DESCRIPTION
**Pull request by mkrze**
Additionally added logging from lm_eval, thanks to this we can see the parameters passed to simple_evaluate function:

`lm_eval.evaluator - INFO - 2025-03-06 11:44:50,953 - Setting random seed to 0 | Setting numpy seed to 1234 | Setting torch manual seed to 1234 | Setting fewshot manual seed to 1234`
`lm_eval.evaluator - INFO - 2025-03-06 11:44:50,953 - Initializing vllm model, with arguments: {'pretrained': '/mnt/weka/data/pytorch/llama3.1/Meta-Llama-3.1-8B-Instruct', 'tensor_parallel_size': 1, 'add_bos_token': True, 'dtype': 'bfloat16', 'max_model_len': 4096, 'max_num_seqs': 128, 'trust_remote_code': False, 'enable_lora': True, 'lora_local_path': '/mnt/weka/data/pytorch/llama3/Lora-Adapter-Llama-3-8B-Instruct'}`

### CODE REVIEW

File: .jenkins/lm-eval-harness/test_lm_eval_correctness.py [hunk 1]
```diff
@@ -20,6 +21,19 @@
 
 import vllm
 
+# Configure logging
+logging.basicConfig(level=logging.INFO)
```
Comment (michalkuligowski) [inline .jenkins/lm-eval-harness/test_lm_eval_correctness.py:25]: What new messages are logged?

**Review by michalkuligowski — CHANGES_REQUESTED** [review]
